tools/configure-hostnames.sh

```diff
@@ -7,6 +7,7 @@ declare -a hostnames=(
   "127.0.0.1 iatlas-postgres"
   "127.0.0.1 model-ad-api"
   "127.0.0.1 model-ad-mongo"
+  "127.0.0.1 agora-mongo"
```

Comment:
See comment above the list: keep the list sorted in alphabetical order

Reply:
Done in the next commit